dash.js

```diff
@@ -779,17 +650,17 @@ const myDash = new Lang.Class({
                    !actor.animatingOut;
         });
 
-        let appIcons = iconChildren.map(function(actor){
+        let appIcons = iconChildren.map(function(actor) {
             return actor.child._delegate;
         });
 
       return appIcons;
     },
 
-    _updateAppIconsGeometry: function() {
+    _updateAppIcons: function() {
         let appIcons = this._getAppIcons();
-        appIcons.forEach(function(icon){
-            icon.updateIconGeometry();
+        appIcons.forEach(function(icon) {
+            icon.onWindowsChanged();
         });
```

Comment:
It is necessary to call onWindowsChange here because window stealing could result in changes beyond just geometry. (Without it, there are bugs with window stealing.) I changed the name because I added the onWindowsChange call, and so it was no longer just about geometry.